### DISCUSSION TITLE
When is 2.2.1 release be expected?

### DISCUSSION THREAD

**Discussion by fj-sanchez** [apache/airflow#19311] [General] [answered]
For a few days it has been completed, but more issues seem to be added to it, do you have any idea when the release will be created?

Thanks!

**Comment by potiuk** [comment, marked as answer]
Voting finishes today (I think in about two hours) and we already have enough binding votes - so likely today (see https://lists.apache.org/thread.html/rd797a86982f6b8da723356ccd4b3ead559027967e3f973904b02fb78%40%3Cdev.airflow.apache.org%3E)

**Reply by fj-sanchez** [reply]
Thanks! I had no idea about the mail list :+1:
